tests/test_tracing.py

```diff
@@ -805,6 +805,7 @@ def test_create_inferred_span_from_api_gateway_event(self):
         self.assertEqual(span.span_type, "http")
         self.assertEqual(span.get_tag(InferredSpanInfo.TAG_SOURCE), "self")
         self.assertEqual(span.get_tag(InferredSpanInfo.SYNCHRONICITY), "sync")
+        self.assertEqual(span.get_tag("peer.service"), "function")
```

Comment:
I think it would be good to have standalone tests also, for example here you are assertion just the fallback. I know the implementation is simple but as I mentioned here https://github.com/DataDog/datadog-lambda-python/pull/337#discussion_r1204642506 there seems different resolutions of peer.service

Reply:
Yes, I know.
Nonetheless, since I cannot just set `DD_SERVICE` in `setUp` because `tracing.py` is initializing the variable that stores it. There is no way of setting a new value in the actual state of `tracing.py`. I want to change this in another PR where we refactor the file and how inferred spans are handled.